js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -85,7 +88,9 @@ class RequestsPage extends Component {
       >
         <TxList
           address=''
+          blockNumber={ blockNumber }
           hashes={ localHashes }
+          newError={ newError }
```

Comment:
I kept it like this to avoid confusion. That's a redux function I am passing in. I think it's more obvious to keep the name.